lms/static/styles/_utilities.scss

```diff
@@ -0,0 +1,2 @@
+@use '@hypothesis/frontend-shared/styles/util';
+@use 'tailwindcss/utilities';
```

Comment:
We could put any local `.u-*` classes here if we chose to. We just don't have any right now!